src/Mod/Assembly/UtilsAssembly.py

```diff
@@ -564,8 +566,10 @@ def findClosestPointToMousePos(candidates_points, mousePos):
 
 
 def findVertexNameInObject(vertex, obj):
-    for i, vtx in enumerate(obj.Shape.Vertexes):
-        if vtx.Point == vertex.Point:
+    for i, _ in enumerate(obj.Shape.Vertexes):
+        if (
+            obj.Shape.getElement("Vertex" + str(i + 1)).Point == vertex.Point
+        ):  # Don't use the vtx param from enumerate's 2nd output to avoid hasher warning
             return "Vertex" + str(i + 1)
```

Comment:
How about splitting this up to reduce the amount each line does and reuse the generated vertex name?
```py
        # Don't use the vtx param from enumerate's 2nd output to avoid hasher warning
        vertexName = f"Vertex{i + i}"
        element = obj.Shape.getElement(vertexName)
        if element.Point == vertex.Point:
            return vertexName
```